FiveCalls/FiveCalls/AppState.swift

```diff
@@ -43,16 +53,21 @@ class AppState: ObservableObject, ReduxState {
             
             switch locationType {
             case "address", "zipCode":
-                location = NewUserLocation(address: locationValue, display: locationDisplay)
+                self.location = NewUserLocation(address: locationValue, display: locationDisplay)
             case "coordinates":
                 let locValues = locationValue.split(separator: ",")
                 if locValues.count != 2 { return }
                 guard let lat = Double(locValues[0]), let lng = Double(locValues[1]) else { return }
                 
-                location = NewUserLocation(location: CLLocation(latitude: lat, longitude: lng), display: locationDisplay)
+                self.location = NewUserLocation(location: CLLocation(latitude: lat, longitude: lng), display: locationDisplay)
             default:
                 Logger().warning("unknown stored location type data: \(locationType)")
             }
         }
+        
+        // load the issue completion cache
+        if let plistSupportableIssueCache = UserDefaults.standard.object(forKey: UserDefaultsKey.issueCompletionCache.rawValue) as? [String: [String]] {
+            self.issueCompletion = Dictionary(uniqueKeysWithValues: plistSupportableIssueCache.map({ key, value in (Int(key) ?? 0, value) }))
```

Comment:
should we coalesce to 0 or just skip or error if we can't create a key?

Reply:
Yeah, you're right. I'll drop anything that can't be converted to an Int.